### DISCUSSION TITLE
Retrieve the url of the notice published through the Web Service Publication

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#501] [Q&A] [answered]
We have seen that notices published using eForms can be found from ted search (https://ted.europa.eu/TED/search/search.do).

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/889bd696-371e-4f35-932c-8114c973505e)

These files still have a publication url like "the old files" (for example https://ted.europa.eu/udl?uri=TED:NOTICE:331151-2023:TEXT:EN:HTML&src=0).

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/20154060-0a89-4a43-aa65-605d0ab9b20a)

How can we retrieve the url? The search-esentool operation of the web service publication does not return that information.

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/4afdde52-4b01-4d67-b956-671d9167a3d5)

Thanks in advance

**Comment by AlexAS81** [comment]
Could it be composed using the publicationId?

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/72ce089a-1cf7-44f0-a008-be0f2bc55598)

https://ted.europa.eu/udl?uri=TED:NOTICE: + publicationID + :HTML:EN:HTML

For example:
https://ted.europa.eu/udl?uri=TED:NOTICE:5269-2023:HTML:EN:HTML

Thanks

**Reply by pcolast** [reply, marked as answer]
Hello,

The approach that you wrote is correct. The structure of the URL will be preserved also in the future.

For the current version of the TED website the publicationId in the URL can have max 6 digits, thus if publicationID has 8, you need to remove the leading 00 from the publicationId.

Have a nice day,
Stefan

**Reply by JialvarezMan** [reply]
Hello Stefan,

In regards of this question, what is the URI to see the published notices in preview on development enviroment? Because the one that we are talking here is the URI of Production.

Thanks in advance.

**Reply by pcolast** [reply]
Hello,

The TED website is not available in the "preview" environment. 

Have a nice day,
Stefan

**Reply by JialvarezMan** [reply]
Ok, thanks for the fast reply.
